src/Auth_Command.php

```diff
@@ -265,11 +269,28 @@ private function generate_global_auth_files() {
 	 * Generates auth files for a site
 	 *
 	 * @param string $site_url URL of site
+	 * @param string $site_type Type of site: wp|subdir|subdom
 	 *
 	 * @throws Exception
 	 */
-	private function generate_site_auth_files( string $site_url ) {
-		$site_auth_file = EE_ROOT_DIR . '/services/nginx-proxy/htpasswd/' . $site_url;
+	private function generate_site_auth_files( string $site_url, string $site_type ) {
+
+		$auth_file_name = '';
+		switch ( $site_type ) {
+			case 'wp':
+				$auth_file_name = $site_url;
+				break;
+			case 'subdir':
+				$auth_file_name = $site_url;
+				break;
+			case 'subdom':
+				$auth_file_name = '*.' . $site_url;
```

Comment:
For subdom site, two files will have to be generated:
1. `$site_url`
2. `*.$site_url`